dkms.in

```diff
@@ -995,23 +995,59 @@ clean_build()
     rm -rf "$dkms_tree/$module/$module_version/build"
 }
 
+# Different distros put scripts/sign-file in different places
+locate_sign_file()
+{
+    local kernel_release="$(uname -r)"
+    local debian_location="/usr/lib/linux-kbuild-${kernel_release%.*}/scripts/sign-file"
+    local ubuntu_location="/usr/src/linux-headers-$kernel_release/scripts/sign-file"
+    local fedora_location="/usr/src/kernels/$kenel_release/scripts/sign-file"
```

Comment:
It seems that both paths are usable

    $ /usr/lib/modules/5.17.8-100.fc34.x86_64/build/scripts/sign-file
    Usage: scripts/sign-file [-dp] <hash algo> <key> <x509> <module> [<dest>]
           scripts/sign-file -s <raw sig> <hash algo> <x509> <module> [<dest>]
    $ /usr/src/kernels/5.17.8-100.fc34.x86_64/scripts/sign-file
    Usage: scripts/sign-file [-dp] <hash algo> <key> <x509> <module> [<dest>]
           scripts/sign-file -s <raw sig> <hash algo> <x509> <module> [<dest>]

And they are both contained in the `kernel-devel` package

    $ rpm --query --file /usr/lib/modules/5.17.8-100.fc34.x86_64/build/scripts/sign-file
    kernel-devel-5.17.8-100.fc34.x86_64
    $ rpm --query --file /usr/src/kernels/5.17.8-100.fc34.x86_64/scripts/sign-file
    kernel-devel-5.17.8-100.fc34.x86_64

But looking at the list of files of the `kernel-devel` package reveals that only one of the two is the "canonical" path:

    $ rpm --query --list kernel-devel-5.17.8-100.fc34.x86_64 | grep scripts/sign-file
    /usr/src/kernels/5.17.8-100.fc34.x86_64/scripts/sign-file
    /usr/src/kernels/5.17.8-100.fc34.x86_64/scripts/sign-file.c

Lastly, the [Fedora docs](https://docs.fedoraproject.org/en-US/fedora/latest/system-administrators-guide/kernel-module-driver-configuration/Working_with_Kernel_Modules/#sect-signing-kernel-module-with-the-private-key) also only mention the path under `/usr/src`, so that's probably the one that should be used here too.